src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -799,7 +800,7 @@ namespace winrt::TerminalApp::implementation
 
         // Give term control a child of the settings so that any overrides go in the child
         // This way, when we do a settings reload we just update the parent and the overrides remain
-        TermControl term{ *(winrt::get_self<TerminalSettings>(settings)->CreateChild()), connection };
+        TermControl term{ settings.MakeChild(), connection };
```

Comment:
Is this the same language as Profile uses?

Reply:
I'm a bit confused by your question. `CreateChild` was introduced by `IInheritable`. But we can't use it here because `CreateChild` returns a `com_ptr` and that can't be exposed in the IDL. So `MakeChild` is just a wrapper for `CreateChild` that returns the projected type.

We're doing the same thing here as before, so there's no need to do anything like `Profile` (i.e. worrying about layering JSON).

Reply:
Maybe we should promote "MakeChild" to a static function, "TerminalSettings::CreateInheritedSettings(parent)"? I want to get the name right.

Reply:
Love it! Fits in with all the other static "ctors" we have.